src/size.rs

```diff
@@ -178,6 +178,11 @@ impl Size {
         Size::new(self.width.trunc(), self.height.trunc())
     }
 
+    /// Returns the aspect ratio of a rectangle with the given size.
+    pub fn aspect_ratio(self) -> f64 {
+        self.height / self.width
```

Comment:
I wonder if it's worth handling the zero cases specially.

What are the aspect ratios of `(0, 10)`, `(0, 0)` and `(5, 0)`? If we're returning `NaN` that should probably be documented.

Reply:
I think NaN is probably the correct result of (0, 0), since 0 / 0 really doesn't mean anything. For the others they should just be 0 and infinity.

Reply:
I could imagine (0, 0) being 1? I know this is going to offend everyone, and I'm not going to fight for it. `NaN` is fine. :)

Reply:
to be honest it could be anything, so if 1 causes fewer problems then fine. 0 = 1 x 0 after all. :)

Reply:
I think @raphlinus will get mad if `0, 0` is 1. 😢